lib/dgram.js

```diff
@@ -639,6 +639,25 @@ Socket.prototype.unref = function() {
   return this;
 };
 
+
+Socket.prototype.setRecvBufferSize = function(size) {
+  var err = this._handle.setRecvBufferSize(size);
+
+  if (err) {
+    throw errnoException(err, 'setRecvBufferSize');
+  }
+};
+
+
+Socket.prototype.setSendBufferSize = function(size) {
+  var err = this._handle.setSendBufferSize(size);
```

Comment:
Because `_handle.setSendBufferSize()` includes a `CHECK` that will abort if `size` is not `UInt32`, these should do a proper type check in the js side. Otherwise users will not be happy.